README.md

```diff
@@ -96,6 +94,14 @@ npm start
 
 ## Product Overview (Don V.) <a name="overview"></a>
 
+**Image Gallery Features**
+* *Default Gallery View*: A visual component that provides a way to browse the available photos for each product style. Using the arrows available on either side of the view or clicking the thumbnails indexed on the left of the gallery will allow you to peruse through the photos. Clicking on the view will open an Expanded view of the Image Gallery
+* *Expanded View*: A visual component that is opened by clicking on the default view of the Image Gallery. The component spans across the entire browser width and provides a larger view of the selected photo. Navigation controls at the top of the photo will allow you to continue browsing through photos without having to leave the view. Clicking on the larger photo will allow you to view a "zoomed-in" portion of the photo at it's maximum size. Moving your mouse during the mode will pan the zoomed in photo until the mouse leaves the photo frame, or mouses over the navigation buttons.
+
+**Product DetailsFeatures**
+* *Style Selector*: Each product comes in a variable amount of styles. The style selector lists a visual thumbnail of each available style and allows you to click on the desired style to update the image gallery and stock selection.
```

Comment:
I like "varied" amount of styles better, but I think you could leave it this way.
The style selector lists "visual thumbnails of each..."